docs/enhancements/template.md

```diff
@@ -0,0 +1,154 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@janedoe"
+reviewers:
+  - TBD
+  - "@alicedoe"
+approvers:
+  - TBD
+  - "@oscardoe"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+see-also:
+  - "docs/enhancements/this-other-neat-thing.md"
+replaces:
+  - "docs/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "docs/enhancements/our-past-effort.md"
```

Comment:
Is this supposed to be the inverse of `replaces`?
If that's the case, then shouldn't the doc here be something that happens after this doc? The names are just a bit confusing.

Reply:
removing this to simplify.